managed/models/database.go

```diff
@@ -838,6 +838,13 @@ var databaseSchema = [][]string{
 		FROM services
         WHERE service_type = 'mongodb';`,
 	},
+	79: {
+		`CREATE TABLE files (
```

Comment:
Just a thought -`files` here feels a bit generic (especially since we're only using this for PMM base files, maybe something like `base_files`? or `init_files`? 

I don't know what might be a better name here, just thought it'd be great to be more specific.

Reply:
It is not described in the problem. But it's not just for config files. There may be others in the future.